flink-state-backends/flink-statebackend-forst/src/main/java/org/apache/flink/state/forst/ForStResourceContainer.java

```diff
@@ -432,17 +395,17 @@ private static boolean prepareDirectories(Path basePath, Path dbPath) throws IOE
      * @throws Exception if any unexpected behaviors.
      */
     public void clearDirectories() throws Exception {
-        if (remoteBasePath != null) {
-            forStFileSystem.delete(remoteBasePath, true);
+        if (pathContainer.getRemoteBasePath() != null) {
```

Comment:
nit: use a variable for pathContainer.getRemoteBasePath() and pathContainer.getLocalBasePath()